### PR TITLE
Enabling colors in CLI (Golang)

### LINKED ISSUE #349: Enable colors in Go CLI
Since our Java CLI supports colors and it's quite a useful feature, we also need to support them in Go CLI. 

#### Acceptance criteria
1. Investigate on libraries which provide ANSI colors support in CLI ([Some inspiration](https://github.com/avelino/awesome-go))
2. Enable colors in CLI
3. Make sure that chosen colors are visible in light and dark terminals

### PR DESCRIPTION
**Pull request by maxim-tschumak**
Closes #349.

Implemented in a pair-programming session together with @maschleg :)

### CODE REVIEW

File: cli-go/zally/supported_rules.go [hunk 1]
```diff
@@ -51,5 +52,15 @@ func buildRequest(httpVerb string, path string, context *cli.Context) (request *
 }
 
 func printRule(rule *Rule) {
-	fmt.Printf("%s %s: %s\n\t%s\n\n", rule.Type, rule.Code, rule.Title, rule.URL)
+	colorize := colorizeByTypeFunc(rule.Type)
+	fmt.Printf("%s %s: %s\n\t%s\n\n", colorize(rule.Code), colorize(rule.Type), rule.Title, rule.URL)
```
Comment (netme) [inline cli-go/zally/supported_rules.go]: I like it, should be easy to use.

File: cli-go/zally/supported_rules.go [hunk 2]
```diff
@@ -51,5 +52,15 @@ func buildRequest(httpVerb string, path string, context *cli.Context) (request *
...
+}
+
+func colorizeByTypeFunc(ruleType string) func(interface {}) aurora.Value {
+	switch ruleType {
+	case "MUST": return aurora.Red
+	case "SHOULD": return aurora.Brown
+	case "HINT": return aurora.Green
```
Comment (netme) [inline cli-go/zally/supported_rules.go]: `MAY` is missing

Comment (maxim-tschumak) [inline cli-go/zally/supported_rules.go]: Thanks, I've added `MAY` color.

**Comment by netme** [comment]
:+1:

**Pull request merged**